.github/workflows/nightly-builds.yml

```
@@ -0,0 +1,96 @@
+
+name: openjverein nightly release
+
+on:
+  push:
+    branches: 
+      - master
+    tags-ignore: 
+      - '**'
+  pull_request:
+    types: 
+      - closed
+
+jobs:
+  build:
+    name: Building release and upload to branch
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up JDK 11 for x64
+      uses: actions/setup-java@main
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        architecture: x64
+
+    - name: Check out jameica
+      uses: actions/checkout@main
+      with:
+        repository: willuhn/jameica
+        token: ${{ secrets.SERVICE_ACCOUNT_PAT }}
+        path: jameica
+        
+    - name: Build jameica nightly
+      working-directory: ./
+      run: ant -noinput -buildfile jameica/build/build.xml nightly
+    
+    - name: Check out hibiscus
+      uses: actions/checkout@main
+      with:
+        repository: willuhn/hibiscus
+        token: ${{ secrets.SERVICE_ACCOUNT_PAT }}
+        path: hibiscus
+        
+    - name: Build hibiscus nightly
+      working-directory: ./
+      run: ant -noinput -buildfile hibiscus/build/build.xml nightly
+
+    - name: Checkout openjverein
+      id: openjverein_checkout
+      uses: actions/checkout@main
+      with:
+        path: jverein
+    
+    - name: Build openjverein plugin
+      id: openjverein
+      working-directory: ./
+      run: |
+        ant_output=$(ant -e -q -noinput -buildfile jverein/build/build.xml nightly)
+        echo $ant_output
+
+        ssa="SELECTED_VERSION="
+        ssb=".zip"
+        text="${ant_output#*${ssa}}"
+        text="${text%${ssb}*}.zip"
+        tmp_version=$(echo $text | sed -rn 's/^([^[:blank:]]*).*$/\1/p')
+        tmp_version=$tmp_version
+
+        ssa="SELECTED_FILENAME="
+        text="${ant_output#*${ssa}}"
+        text="${text%${ssb}*}.zip"
+        tmp_filename=$(echo $text | sed -rn 's/^([^[:blank:]]*).*$/\1/p')
+
+        ssa="SELECTED_PATH="
+        text="${ant_output#*${ssa}}"
+        text="${text%${ssb}*}.zip"
+        tmp_path=$(echo $text | sed -rn 's/^([^[:blank:]]*).*$/\1/p')
+
+        echo "selected_version=${tmp_version}" >> $GITHUB_OUTPUT
+        echo "selected_filename=$tmp_filename" >> $GITHUB_OUTPUT
+        echo "selected_path=$tmp_path" >> $GITHUB_OUTPUT
+
+        builddatetime=$(date +'%Y-%m-%d %H:%M')
+        echo "### Version: $tmp_version | filename: $tmp_filename | build datetime: $builddatetime" >> $GITHUB_STEP_SUMMARY
+
+    - name: Create release and publish
+      uses: "marvinpinto/action-automatic-releases@latest"
+      with:
+        repo_token: ${{ secrets.SERVICE_ACCOUNT_PAT }}
+        automatic_release_tag: ${{ steps.openjverein.outputs.selected_version }}
+        prerelease: true
+        title: Release ${{ steps.openjverein.outputs.selected_version }}
+        files: ./jverein/${{ steps.openjverein.outputs.selected_path }}
```

2 / 2

```suggestion
        files: ./jverein/${{ steps.openjverein.outputs.selected_path }}
      secrets: inherit
```

Referenz:

* [Verwenden von Eingaben und Geheimnissen in einem wiederverwendbaren Workflow](https://docs.github.com/de/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow)

---

Ich bin mir aber nicht sicher, ob die Action marvinpinto/action-automatic-releases das unterstützt.